.env.baklava

```diff
@@ -76,7 +77,10 @@ EKSPORTISTO_DOCKER_IMAGE_TAG="b199a634866ef34c4fa1925381dff4de40eb0a3c"
 EKSPORTISTO_SUFFIX='1'
 
 ATTESTATION_SERVICE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/celo-monorepo"
-ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-dc5e5dfa07231a4ff4664816a95eae606293eae9"
+ATTESTATION_SERVICE_DOCKER_IMAGE_TAG="attestation-service-1-0-5"
+
+# We only configured 15 nexmo applications. So delivery receipts from the second 15 will always go to the wrong place for now
```

Comment:
haha ok -- how much is the monthly fee per application.  do we gain much from having half use their own application?